ios/brave-ios/Sources/Brave/Frontend/Browser/Playlist/Controllers/PlaylistLegacyCarplayController.swift

```diff
@@ -27,7 +27,7 @@ enum PlaylistCarplayError: Error {
   case itemExpired(id: String)
 }
 
-class PlaylistCarplayController: NSObject {
+class PlaylistLegacyCarplayController: NSObject {
```

Comment:
Eventually we can delete them, right now new playlist will sit side-by-side with old playlist behind a feature flag so that design + product can get their hands on it and we can adjust it. New playlist can basically be considered "beta" right now without affecting any of the old playlist logic